test/unit/module/real/core/bit_andnot.cpp

```diff
@@ -43,32 +43,72 @@ EVE_TEST_TYPES( "Check return types of bit_andnot"
 //  bit_andnot tests
 //==================================================================================================
 EVE_TEST( "Check behavior of bit_andnot on integral types"
-          , eve::test::simd::integers
-          , eve::test::generate ( eve::test::randoms(eve::valmin, eve::valmax)
-                                , eve::test::randoms(eve::valmin, eve::valmax)
-                                )
+            , eve::test::simd::integers
```

Comment:
please fix the settings of your text editor to not generates those useless changeset

Reply:
idem